Formula/neovim.rb

```diff
@@ -20,12 +21,13 @@ class Neovim < Formula
 
   depends_on "cmake" => :build
   depends_on "luarocks" => :build
+  depends_on "luv" => :build
```

Comment:
Based on the conversation at https://github.com/neovim/neovim/issues/13847#issuecomment-770209466 , should `ninja` be added here as a build dependency? Thanks.

Reply:
It's not really necessary. Ninja's selling point is that it's supposed to be faster than Make, but that's only really true for very large projects that take a long time to (re-)compile. Neovim takes about a minute, so there's no need to add an extra dependency here.